base/array.jl

```diff
@@ -2,6 +2,38 @@
 
 ## array.jl: Dense arrays
 
+"""
+    Array{T}(dims)
+    Vector{T}(m)
+    Matrix{T}(m, n)
+
+`Array{T}(dims)` constructs an uninitialized dense array with element type `T`. `dims` may
+be a tuple or a series of integer arguments, each giving the size of the corresponding
+dimension. `Vector{T}(m)` and `Matrix{T}(m, n)` are stricly equivalent aliases to create one-
+and two-dimensional arrays.
+
+The syntax `Array(T, dims)` is also available, but deprecated.
+
+
```

Comment:
Trying to associate two docstrings with the same signature should print a warning and only store the second. I suppose you're getting the undefined error there since during bootstrap it isn't defined. I'll fix that up tomorrow.

Best to attach each one to a more specific signature if possible.